### ISSUE TITLE
[FEATURE]: Show usage of SAP cloud SDK with Azure functions

### ISSUE THREAD

**Issue by MartinPankraz** [Azure-Samples/app-service-javascript-sap-cloud-sdk-quickstart#23]
**Describe the solution you'd like**
Usage of SAP Cloud SDK with Azure app service is great. Would like to get insights how to apply to Azure functions too.

**Describe alternatives you've considered**
Use the OData SDK [here](https://github.com/Azure/azure-sdk-for-sap-odata)

**Comment by lechnerc77** [comment]
As you have a UI attached to the setup with the Azure App Service, would it make sense to have a bit of a bigger scope and enhance it with Azure Static Web  Apps and Azure Functions?

**Comment by MartinPankraz** [comment]
Yes, I would like that. The web apps can be covered here for sure, but should we separate the function in its own repos? Or increase scope as you suggested. Would it still serve as quickstart though?

**Comment by lechnerc77** [comment]
That's a very valid point. 
My 2 cent: 

- I would keep things in one repository to keep it self-contained and in sync (at least I am more likely to adjust both projects in sync if they are in the same repo :-) ). That would probably cause a bit of restructuring to get both "flavors" in and not to clutter the repo.
- To make the content easier to digest I would introduce [workspaces](https://code.visualstudio.com/docs/editor/workspaces) that allow to group the relevant artifacts for one or the other case and display them separately in VS Code and GH Codespaces (seems to work there too see [here](https://github.com/community/community/discussions/6843)).

From the top of my head the repo would then have the generic files in the root and we then have to folders, one for the Web App and one for the Static Web App. 
In addition we introduce a workspace definition that allow to open the files (maybe even in different layout) in a dedicated workspace.

**Comment by MartinPankraz** [comment]
agreed. Like the solution with workspaces to isolate the topics but still profit from the same foundation. Would also be fine to put functions there too, since the logical isolation is still there.
